docs/rst/introduction/metapackages.rst

```diff
@@ -1,9 +1,9 @@
-.. _vulcanexus_binary_distribution:
+.. _vulcanexus_metapackages:
 
-Vulcanexus binary distributions
-===============================
+Vulcanexus Metapackages
+=======================
 
-*Vulcanexus* provides several binary distributions that install specific packages depending on the developer's needs.
+*Vulcanexus* provides several installations with specific packages depending on the developer's needs.
```

Comment:
I'd not change the binary distributions here as it is what it is. Maybe a clarification of what a binary distribution is can be added here.

Reply:
But I don't think it's necessary to add such clarification, as whoever is not sure about the concept should probably understand after skimming the page and seeing all the `apt install` commands in it.